### DISCUSSION TITLE
Maybe it is better to remove @author from default java templates?

### DISCUSSION THREAD

**Discussion by PavelTurk** [apache/netbeans#8056] [Ideas]
I thought about it for a while, then I read https://stackoverflow.com/questions/17269843/javadoc-author-tag-good-practices .
Maybe it is better to remove @author from default java templates?

**Comment by wumpz** [comment]
What about introducing a subtemplate "initial javadoc" like the licence part? So everybody is able to modify it to their own code style. IMHO modifing, especially adding to templates (with logic) is quite difficult, removing not.
